migrations/2020-05-19-132810_create_chunks/up.sql

```diff
@@ -1,12 +1,12 @@
 -- Your SQL goes here
 CREATE TABLE chunks (
-	block_id numeric(45) NOT NULL,
-	hash varchar(58) PRIMARY KEY NOT NULL,
-	shard_id numeric(20) NOT NULL,
+	block_id numeric(45) NOT NULL, -- numeric(precision) 45 digits should be enough to store u128::MAX
```

Comment:
`ChunkHeaderView` has `prev_block_hash` but doesn't have `block_hash` in has `height_created` and `height_included` and we connect chunks to blocks by current block being processed by indexer at the moment.

Why break it?

Reply:
Break it for consistency and sanity. When a chunk is produced, the block does not exist, so there is a reason why block hash cannot be there, but since we observe chunks inside blocks, we already know the block hash.